### PR TITLE
Add support for bank on eps/p24

### PR DESCRIPTION
**Pull request by richardm-stripe**
r? @brandur-stripe 
cc @stripe/api-library-reviewers 

Add support for `Bank` for eps and p24 on Charge and PaymentMethod.
Add support for `TOSShownAndAccepted` on `PaymentMethodP24Params`

### CODE REVIEW

File: paymentmethod.go [hunk 1]
```diff
@@ -164,7 +165,8 @@ type PaymentMethodOXXOParams struct {
 // PaymentMethodP24Params is the set of parameters allowed for the `p24` hash when creating a
 // PaymentMethod of type P24.
 type PaymentMethodP24Params struct {
-	Bank *string `form:"bank"`
+	Bank                *string `form:"bank"`
+	TOSShownAndAccepted *bool   `from:"tos_shown_and_accepted"`
```
Comment (richardm-stripe) [inline paymentmethod.go:169]:
```suggestion
	TOSShownAndAccepted *bool   `form:"tos_shown_and_accepted"`
```

File: paymentintent.go [hunk 2]
```diff
@@ -172,10 +172,12 @@ type PaymentIntentPaymentMethodDataParams struct {
 	AUBECSDebit    *PaymentMethodAUBECSDebitParams `form:"au_becs_debit"`
 	BillingDetails *BillingDetailsParams           `form:"billing_details"`
 	Card           *PaymentMethodCardParams        `form:"card"`
+	EPS            *PaymentMethodEPSParams         `form:"eps"`
```
Comment (brandur-stripe) [inline paymentintent.go:175]: Arg, it looks like this should be "EPS", but it's called `Eps` on `Charge`. We should probably keep it the same between the two just for the sake of consistency.

**Review by brandur-stripe — COMMENTED** [review]
Thanks Richard. Left one comment, but LGTM.

**Comment by brandur-stripe** [comment]
ptal @richardm-stripe

File: paymentintent.go (hunk 2 above)
Comment (richardm-stripe) [reply paymentintent.go:175]: It's Eps on Charge, but unfortunately it is also EPS in paymentmethod.go and account.go and I think it is probably better to be consistent with that?
ptal @brandur

Comment (brandur-stripe) [reply paymentintent.go:175]: > It's Eps on Charge, but unfortunately it is also EPS in paymentmethod.go and account.go and I think it is probably better to be consistent with that?

Ah, fair. Yep, let's go with `EPS` then.

**Review by brandur-stripe — APPROVED** [review]

**Pull request merged**